app/components/svg-image.jsx

```diff
@@ -1,4 +1,21 @@
-import React, {PropTypes} from 'react';
+import React from 'react';
+
+const FILTERS = {
+    invert: "url('#svg-invert-filter')",
+}
+
+const INVERT =
+  '<svg style="position: fixed; right: 100%; top: 100%; visibility: hidden;">\
```

Comment:
Lint is complaining `Multiline support is limited to browsers supporting ES5 only.`, which might not be a big deal. Backticks will remove this warning, however.